example/bwmf/demo/run_demo.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/env bash
+
+# saving current dir
+pushd $(pwd)
+
+# goto script dir and stack it, we'll be back later
+pushd $(dirname $0)
```

Comment:
Not exactly the same. It's for executing the script from any path. 
./run_demo.sh works, example/bwmf/demo/run_demo.sh also works.